package.json

```diff
@@ -8,7 +8,15 @@
     "react",
     "ssr"
   ],
-  "main": "lib/index.js",
+  "main": "./lib/cjs/cjs-relay-hooks.js",
+  "module": "./lib/es-relay-hooks.mjs",
+  "exports": {
+    ".": {
+      "import": "./lib/es-relay-hooks.mjs",
+      "require": "./lib/cjs/cjs-relay-hooks.js"
+    },
+    "./package.json": "./package.json"
+  },
```

Comment:
Starting with Node.js 14 you cannot simply import anything from a package. You can only use the exports declared in the `exports` map.

E.g. `import { fetchResolver } from "relay-hooks/lib/FetchResolver";` would fail. If that should be possible a entry for each file must be declared in the exports map.

Reply:
hi @n1ru4l,
i tried this package in react-relay-offline and i get this error.
```
react-relay-offline\node_modules\relay-hooks\lib\QueryFetcher.js:1
    ({"Object.<anonymous>":function(module,exports,require,__dirname,__filename,global,jest){import * as areEqual from 'fbjs/lib/areEqual';
    SyntaxError: Cannot use import statement outside a module
```
Did you mean this?

Reply:
If you are referring to this: https://github.com/morrys/react-relay-offline/blob/c12787d3d0c3325389cd4c8d06ea2f3ffdac800a/src/runtime/loadQuery.ts#L3

Then yes, instead this file should be explicitly exposed via the exports map. See https://nodejs.org/api/packages.html#packages_package_entry_points

Is there any specific reason why you don't re-export those files from the entry point?

Reply:
Also related to https://github.com/relay-tools/relay-hooks/pull/161#discussion_r634938524